metrics/web.go

```diff
@@ -0,0 +1,141 @@
+package metrics
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	slowBuckets = []float64{
+		1, 30, 60, 120, 180, 300, 600, 900, 1200, 1800, 2700, 3600, 7200, 18000, 36000,
```

Comment:
ooh yeah, this bad boy here ends up generating 16 labels for each measurement that we end up adding 😅 moving to summaries, that would reduce quite a bit the load we put on the server (specially given that `slowBuckets` is used exclusively for `build` time - the metric with the highest cardinality).